src/applications/vre/28-8832/config/form.js

```diff
@@ -7,7 +7,7 @@ import IntroductionPage from '../containers/IntroductionPage';
 import ConfirmationPage from '../containers/ConfirmationPage';
 import { statusSelection } from './chapters/status-selection';
 import { veteranInformation } from './chapters/veteran-information';
-import GetFormHelp from '../components/GetFormHelp';
```

Comment:
unrelated work but a quick fix. 28-8832 was not using the correct component for form footer.